parachain/pallets/omni-account/src/lib.rs

```diff
@@ -111,14 +176,56 @@ pub mod pallet {
 
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
+		// dispatch the `call` as RawOrigin::OmniAccount
 		#[pallet::call_index(0)]
 		#[pallet::weight((195_000_000, DispatchClass::Normal))]
+		pub fn dispatch_as_omni_account(
+			origin: OriginFor<T>,
+			account_hash: H256,
+			call: Box<<T as Config>::RuntimeCall>,
+		) -> DispatchResult {
+			let _ = T::TEECallOrigin::ensure_origin(origin)?;
+			let omni_account =
+				LinkedIdentityHashes::<T>::get(account_hash).ok_or(Error::<T>::IdentityNotFound)?;
```

Comment:
Does this mean that `who` sent to `link_identity` is the OmniAccount? So, the OmniAccount should exist before linking any account, right?

Reply:
I mean `who` should be the omni-account even in the current impl